bitnami/minio/Chart.yaml

```diff
@@ -24,4 +24,4 @@ maintainers:
 name: minio
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/minio
-version: 12.6.4
+version: 12.6.5
```

Comment:
Could you please rebase your branch and bump the chart version again? Thanks!

Reply:
Sure! I have just updated the branch.